test/services/ProcessCsvServiceSpec.scala

```diff
@@ -112,8 +114,8 @@ class ProcessCsvServiceSpec
     }
 
     "process a row and return an empty list if there are no errors" in {
-      val source = convertToAkkaSource(new File(System.getProperty("user.dir") + "/test/resources/copy/CSOP_OptionsGranted_V4.csv"))
-      val dataValidator = new DataValidator(ConfigFactory.load.getConfig("ers-csop-granted-validation-config"))
+      val source = convertToAkkaSource(new File(System.getProperty("user.dir") + "/test/resources/copy/CSOP_OptionsGranted_V5.csv"))
```

Comment:
Lots of these tests you replace V4 with V5, would it be worth "duplicating" the tests in a similar way to how you did in `test/services/DataGeneratorSpec.scala`? or given that both schemas have identical validation logic and just the row name has changed this is sufficent?

Reply:
as discussed, leaving as is